### PR TITLE
Start making Equipment Table translatable

### PR DESCRIPTION
**Pull request by Jazzis18**
The first few days I worked hard on the code, but then... Unfortunately, the insidious procrastination teaming up with deadlines at work defeated me. So I went missing D:

Fortunately, I was able to find the time after work today to commit changes and create a PR.

### CODE REVIEW

File: equipment_table_creator.html [hunk 1]
```diff
@@ -58,171 +58,181 @@
...
-        textarea_intro:        "This textarea contains code to produce a Wikia table showing",
-        all_quests:            "all quests",
-        quest_lines:           "only quests that are in quest lines",
-        easy_quests:           "only easy quests",
-        medium_quests:         "only medium quests",
-        hard_quests:           "only hard quests",
-        textarea_paste:        "Paste this code into this template:",
+        ///////////////////////////////////////////////////////////////////////////////
+
+        kickstarter: "Kickstarter",
```
Comment (Jazzis18) [inline equipment_table_creator.html]: Here was a link to a Backers article, which is now a redirect to a Kickstarter article. So I fixed it.

**Comment by Alys** [comment]
Thanks very much for this! I'll test it this weekend and deploy it as soon as I can.

**Comment by Jazzis18** [comment]
I think there is still work to do. For example, I could not adequately translate the second sentence of the introduction of this page. Maybe you have any ideas?

File: equipment_table_creator.html [hunk 2]
```diff
@@ -235,8 +245,18 @@
...
+        independent_item: /Independent Item/i,
+        set: /(.+ Set) \(Item ([0-9]+) of ([0-9]+)/i,
+        unofficial_set: /(Floppy<br \/>Hat|Cat<br \/>Hat)/i,
     },
     'ru': {
+        increases: " Увеличивает ",
+        armoire: " Зачарованный сундук: ",
+        independent_item: /Независимый предмет/i,
+        set: /(.+ [Нн]абор|[Нн]абор .+) \(предмет ([0-9]+) из ([0-9]+)/i,
+        unofficial_set: /(широкополая<br \/>шляпа|Шапка(.+)кота)/i,  // XXX_JAZZIS FIXME
```
Comment (Jazzis18) [inline equipment_table_creator.html]: An outdated comment I left for myself. I forgot to remove it.

Comment (Alys) [reply equipment_table_creator.html]: Ha, no worries I'll remove that. :) I'll be looking through all the changes and making any small adjustments that are needed before I deploy it, and/or I'll discuss future changes with you.

**Comment by Alys** [comment]
This is awesome!! Thanks so much for doing all that!

I've made some minor adjustments (https://github.com/Alys/tools-for-habitrpg/commit/5bc3618e552a5e306c86061cd55b931c51a2233c?w=1) and uploaded it: https://oldgods.net/habitrpg/equipment_table_creator.html / https://oldgods.net/habitrpg/equipment_table_creator.html?language=ru

I still haven't fixed the second sentence in the introduction but I'll work on that as soon as I can - definitely by this time next week but hopefully a lot sooner. :)

I've awarded you Tier 5 for your help with this. Congratulations and thank you. :)

**Comment by Alys** [comment]
I've changed the first paragraph to (hopefully) make it easier to translate, and also removed some near-duplicate strings for simplicity.
Here's the changes: https://github.com/Alys/tools-for-habitrpg/commit/e32c52f212c5cdac4f161fee7a04492ab8eefa85

https://oldgods.net/habitrpg/equipment_table_creator.html?language=ru

Let me know if that doesn't work for you!

**Pull request merged**